packages/config/config/devices/0x0267/10002081-13x.json

```diff
@@ -0,0 +1,190 @@
+{
+	"manufacturer": "SimonTech S.L.U",
+	"manufacturerId": "0x0267",
+	"label": "10002081-13x",
+	"description": "Simon IO: Master Roller Blind",
+	"devices": [
+		{
+			"productType": "0x0005",
+			"productId": "0x01c3"
+		},
+		{
+			"productType": "0x0005",
+			"productId": "0x00f4"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Association LED",
+			"$import": "~/templates/master_template.json#base_enable_disable_255"
+		},
+		{
+			"#": "12",
+			"label": "LED Indicator Idle Brightness",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "20%",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Buttons Lock",
+			"$import": "~/templates/master_template.json#base_enable_disable_255"
+		},
+		{
+			"#": "15",
+			"label": "Reset to Default",
+			"valueSize": 2,
+			"minValue": 17170,
+			"maxValue": 39015,
+			"defaultValue": 39015,
+			"writeOnly": true,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Parameters, Groups, and Z-Wave status are restored to default.",
+					"value": 39015
+				},
+				{
+					"label": "Parameters, except Lock long press are restored to default.",
+					"value": 17170
+				}
+			]
+		},
+		{
+			"#": "19",
+			"label": "Short Press Action",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 7,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Toggle on/off",
+					"value": 0
+				},
+				{
+					"label": "Turn on",
+					"value": 1
+				},
+				{
+					"label": "Turn off",
+					"value": 2
+				},
+				{
+					"label": "Left button sends off, right button sends on",
+					"value": 3
+				},
+				{
+					"label": "Turn on, long press to turn off",
+					"value": 4
+				},
+				{
+					"label": "Left button sends off value (parameter 25), right button sends on value (parameter 26)",
+					"value": 6
+				},
+				{
+					"label": "Follows instruction manual",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "20",
+			"label": "Identify",
+			"description": "LED Indicator blinks for 5 seconds to help identify the device.",
+			"$import": "~/templates/master_template.json#base_enable_disable_255",
+			"writeOnly": true
+		},
+		{
+			"#": "21",
+			"label": "Device Mode",
+			"valueSize": 2,
+			"minValue": 176,
+			"maxValue": 177,
+			"defaultValue": 176,
+			"readOnly": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "B0 (0x01 represents load is on)",
+					"value": 176
+				},
+				{
+					"label": "B1 (0xFF represents load is on)",
+					"value": 177
+				}
```

Comment:
I'm not happy with this one. `0xB0` is 176, `0xB1` is 177, so this is likely not meant to be the actual label.
I wonder what/how the `0x01/0xff` is expressed - this could probably serve as a better label.